test/client.test.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+/* eslint-disable */
```

Comment:
Is it possible to disable only a certain check and not whole eslint?

Reply:
Woops my bad, missed this!